pages/linux/dockerd.md

```diff
@@ -0,0 +1,20 @@
+# dockerd
+
+> A persistent process to start and manage docker containers.
+> More information: <https://docs.docker.com/engine/reference/commandline/dockerd/>.
+
+- Run docker daemon and config it to listening to specific sockets:
+
+`dockerd --host {{socket1}} --host {{socket2}}`
+
+- Run docker daemon with specific daemon PID file:
+
+`dockerd --pidfile {{path/to/pid_file}}`
+
+- Run docker daemon in debug mode:
+
+`dockerd --debug={{boolean}}`
+
+- Run docker daemon, set it to log only specific log level or more severe events:
```

Comment:
A log_level always logs events equal to that or more severe. So we can say "Set a specific log level"